src/AspNet.Security.OAuth.DingTalk/DingTalkAuthenticationOptions.cs

```diff
@@ -0,0 +1,51 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using Microsoft.AspNetCore.Http;
+using static AspNet.Security.OAuth.DingTalk.DingTalkAuthenticationConstants;
+
+namespace AspNet.Security.OAuth.DingTalk
+{
+    /// <summary>
+    /// Defines a set of options used by <see cref="DingTalkAuthenticationHandler"/>.
+    /// </summary>
+    public class DingTalkAuthenticationOptions : OAuthOptions
+    {
+        public DingTalkAuthenticationOptions()
+        {
+            ClaimsIssuer = DingTalkAuthenticationDefaults.Issuer;
+            CallbackPath = new PathString(DingTalkAuthenticationDefaults.CallbackPath);
+
+            AuthorizationEndpoint = DingTalkAuthenticationDefaults.AuthorizationEndpoint;
+            UserInformationEndpoint = DingTalkAuthenticationDefaults.UserInformationEndpoint;
+            TokenEndpoint = DingTalkAuthenticationDefaults.TokenEndpoint;
+
+            Scope.Add("snsapi_login");
+
+            ClaimActions.MapJsonKey(ClaimTypes.Name, "nick");//用户在钉钉上面的昵称
+            ClaimActions.MapJsonKey(ClaimTypes.NameIdentifier, "openid");
+            ClaimActions.MapJsonKey(Claims.UnionId, "unionid");//用户在当前开放应用所属企业内的唯一标识
+            ClaimActions.MapJsonKey(Claims.OpenId, "openid");//用户在当前开放应用内的唯一标识
+            ClaimActions.MapJsonKey(Claims.DingId, "dingId");
+            ClaimActions.MapJsonKey(Claims.MainOrgAuthHighLevel, "main_org_auth_high_level");
+        }
+
+        public string AppId
```

Comment:
Is there a strong enough need to create these aliases over the top of standard OAuth terminology?